### ISSUE TITLE
Schedule events get confused by timezones

### ISSUE THREAD

**Issue by Nordicdesign** [cj123/assetto-server-manager#356]
Our server is in Europe (CEST) and we're in the UK (BST). I've used the scheduled events in the past and it worked well, being scheduled on BST and starting when expected. But I've encounter something funny right now:

Time when I tried this: 17:20BST / 18:20CEST
Using my computer's browser in BST (as opposed to connected directly to the server which is CEST)

I schedule an event to start at 17:40 --> Event starts automatically, like it was set in the past. Top right shows championship in progress
I schedule an event to start at 18:40 --> Event gets scheduled at 18:40 BST

I tried several times to make I didn't get the date wrong, always with the same results. 
Thinking about it, maybe it's not possible to schedule an event in less than an hour? This is all due to the server restarting to install updates, always the worst time :D

**Comment by Hecrer** [comment]
Interesting, we have updated how the timezone is worked out, perhaps there's an issue there. Will look into it.

**Comment by WraithStar** [comment]
Well Sol start time setting gets broken by timezones, but I'm not sure it's directly related with this.
I noticed that the server's timezone is UTC, Sol time gets applied properly tho.

**Comment by Hecrer** [comment]
Yeah, we have a little bit of code that tries to get your timezone and convert it to something that Sol expects. But that shouldn't have any impact on event scheduling.
